src/components/utils.ts

```diff
@@ -762,3 +762,10 @@ export function cacheable<Target, Value, Arguments extends unknown[] = unknown[]
 
   return descriptor;
 };
+
+/**
+ * True if screen has mobile size
+ */
+export function isMobile(): boolean {
```

Comment:
I suggest to rename the function as it doesn't return what expected. Something like `isNarrowScreen`

Reply:
renamed to `isMobileScreen`